src/messages/Text/Text.tsx

```diff
@@ -0,0 +1,10 @@
+import { FC } from "react";
+import { MessagePasstroughProps } from "../types";
+
+const Text: FC<MessagePasstroughProps> = props => {
+	const content = props.message.text;
+
+	return <div dangerouslySetInnerHTML={{ __html: content }}></div>;
```

Comment:
I think this has to be optional, at least there was a webchat setting to enable/disable

Reply:
Currently, not. We do it always (with escaping), and we give an option to disable the escaping, that's all :)

Reply:
Well that means we have to provide the same html sanitization and opt-out options. The question being if this should be a global feature of the chat-components. I guess so? with the goal that everything is displayed the same across different products eventually